cmake/ecbuild_get_test_data.cmake

```diff
@@ -106,7 +113,8 @@ endfunction()
 #                          [ DIRLOCAL <dir> ]
 #                          [ MD5 <hash> ]
 #                          [ EXTRACT ]
-#                          [ NOCHECK ] )
+#                          [ NOCHECK ] 
+#                          [ INSECURE_CURL ])
```

Comment:
Same as above, the name ought to be `INSECURE`

Reply:
Fixed.